### ISSUE TITLE
"selector" seems not supporting $exists, $ne ...

### ISSUE THREAD

**Issue by VansonLeung** [aldeed/meteor-tabular#236]
Being an admin:

```
  selector: function(userId) {
    if (Roles.userIsInRole(userId, ["superadmin", "admin"])) { return { _id: {$ne: null} }; }
    if (Roles.userIsInRole(userId, "merchant")) { return { owner: userId, removed: false }; }
  },
```

Expected: admin should show all items
Results: admin show nothing

```
  selector: function(userId) {
    if (Roles.userIsInRole(userId, ["superadmin", "admin"])) { return { _id: {$exists: true} }; }
    if (Roles.userIsInRole(userId, "merchant")) { return { owner: userId, removed: false }; }
  },
```
Expected: admin should show all items
Results: admin show nothing

**Comment by lucasfurlani** [comment]
@VansonLeung have you tried to enclose "$ne" and "$exists" in double quotes?

Something like this:
```{ return { _id: {"$exists": true} }; ```

It might not be the problem but it worked for me in some cases.

**Comment by soitech** [comment]
Oddly, $exists works fine for me, but $not throws the unknown top level operator exception.

    return {
        $and: [
          {site:usrSite}, 
          {$or: [{ group:''},{group:undefined},{group:{$exists:false}},{group:{$in:usrGroups}}]},
          {$not:{status:{pending:true}}}
        ]
      };

Adding double quotes doesn't make it work.

**Comment by aldeed** [comment]
@soitech Your query syntax is wrong. Like it says, it's not a top level operator.

@VansonLeung Is this still an issue? @soitech says $exists works.

The only thing I can think of is it might get weird when it combines your query with other security and search-box queries. But I would think it would work.

**Comment by soitech** [comment]
Uhhh, yup. That was a terrible example on my part. At that time I replaced that line with 
    {$or:[{'status.pending':false},{'status.pending':{$exists:false}}]}
So again, $exists worked fine. Eventually, with a fresher perspective, I went with the straightforward
    {status:{$ne:{pending:true}}}
So yeah, $exists seems to work fine. $ne works for checking field values, and I can't confirm whether $not works, since I was using it all wrong, but it probably works fine when used correctly to check an operator expression. 

In short, I'm not seeing any problems with query operators in selectors.

**Comment by aldeed** [comment]
Cool. Closing pending a full reproduction.
